src/test/java/org/commonwl/view/PageControllerTest.java

```diff
@@ -19,8 +19,8 @@
 
 package org.commonwl.view;
 
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
```

Comment:
This is part of JUnit4 -> JUnit 5, which is a transitive dependency of SpringBoot.